cvat/apps/documentation/user_guide.md

```diff
@@ -935,7 +1013,56 @@ in/out (when scrolling the mouse wheel) and move (when clicking the mouse
 wheel and moving the mouse) while drawing. At the end of Auto Segmentation,
 a shape is created and you can work with it as a polygon.
 
-![](static/documentation/images/gif009.gif)
+![](static/documentation/images/gif009_DETRAC.gif)
+
+## Auto annotation
+
+1.  First you need to upload deep learning (DL) models using model manager.
+    Only models in OpenVINO™ toolkit format are supported.
+    If you would like to annotate a task with a custom model please convert it
+    to the intermediate representation (IR) format via the model optimizer tool.
+    See [OpenVINO documentation](https://software.intel.com/en-us/articles/OpenVINO-InferEngine) for details.
+
+    ![](static/documentation/images/image099.jpg)
+
+1.  Enter model name, and select model file using "Select files" button. To annotate a task with a custom model
+    you need to prepare 4 files:
+    - ``Model config`` (*.xml) - a text file with network configuration. 
+    - ``Model weights`` (*.bin) - a binary file with trained weights. 
+    - ``Label map`` (*.json) - a simple json file with label_map dictionary like an object with 
+    string values for label numbers.
+    - ``Interpretation script`` (*.py) - a file used to convert net output layer to a predefined structure 
+    which can be processed by CVAT. 
+    More about creating model files can be found [here](https://github.com/opencv/cvat/tree/develop/cvat/apps/auto_annotation).
+
+    ![](static/documentation/images/image104.jpg)
+
+1.  After downloading a model you have to  create a task or find an already created one and
+    click ``Run Auto Annotation`` button.
+
+    ![](static/documentation/images/image119.jpg)
+
+1.  In dialog window select a model you need. If it's necessary select the ``Delete current annotation`` checkbox.
+    Adjust the labels so that the task labels will correspond to the labels of the DL model.
+    Click ``Start`` to begin the auto annotatiton process.
+
+    ![](static/documentation/images/image120.jpg)
+
+1.  At runtime, you can see percentage of completion. You can also cancel the auto annotation
+    process by clicking ``Cancel Auto Annotation``
+
+    ![](static/documentation/images/image121.jpg)
+
+1.  As a result, you will get an annotation with separate bounding boxes (or other shapes)
+
+    ![](static/documentation/images/gif014_DETRAC.gif)
+
+1.  Separate bounding boxes can be edited by removing false positives, adding unlabeled objects, and
+    merged into tracks using ``Merge Shape``
+
+    ![](static/documentation/images/gif015_DETRAC.gif)
+
+Learn more about auto annotations [here](https://github.com/opencv/cvat/tree/develop/cvat/apps/auto_annotation).
```

Comment:
This link have been already mentioned above twice. I am not sure it is needed again.